src/app/medical-record/medical-record.component.ts

```diff
@@ -89,11 +89,21 @@ export class MedicalRecordComponent implements OnInit {
       if (this.participant.data == undefined || this.participant.data.status.indexOf( Statics.EXITED ) == -1) {
         this.participantExited = false;
       }
+      this.pdfs = new Array<PDFModel>();
```

Comment:
Instantiation of `this.pdfs` from this line to line 103 in separate method something like `setInitialPdfs` would make this part of code more understandable